vm/src/obj/objint.rs

```diff
@@ -576,7 +576,7 @@ impl PyInt {
 
     #[pymethod(name = "__sizeof__")]
     fn sizeof(&self, _vm: &VirtualMachine) -> usize {
-        size_of::<Self>()
+        (size_of::<Self>() + (self.value.bits() + 7) & !7) / 8
```

Comment:
Divided by 8 to return bytes instead of bits.

Reply:
`mem::size_of` actually does return bytes, it's the `BigInt::bits` method that returns bits.